cpp/ql/test/library-tests/ir/ir/ir.cpp

```diff
@@ -1690,4 +1690,10 @@ void captured_lambda(int x, int &y, int &&z)
     };
 }
 
+int global_1;
+
+int global_2 = 1;
+
+const int global_3 = 2;
```

Comment:
We probably shouldn't create IR for anything as trivial as `static const int x = 5;`. Maybe only create it if the variable is every used as a glvalue? Or only if the variable is either non-`const` or has a non-constant initializer?

Reply:
This is what the `TODO` in `varHasIRFunc` was about. I was originally planning to include constants in the IR because that means they'll be available as dataflow sources once I add flow through global variables in a followup PR.